doc/nrf/drivers/entropy_cc310.rst

```diff
@@ -7,18 +7,19 @@ CC310 entropy driver
    :local:
    :depth: 2
 
-You can use the CC3xx entropy driver (entropy_cc310) to generate random data using the Arm CryptoCell CC310 hardware.
-This hardware is available on the nRF52840 and nRF9160 devices.
+You can use the CC3xx entropy driver (*entropy_cc310*) to generate random data using the Arm CryptoCell CC310 hardware available on the following Nordic SoCs:
 
-When used on nRF52840, the entropy_cc3xx driver gathers entropy by using the CC310 hardware through the :ref:`nrf_cc3xx_platform_readme`.
+* nRF52840
+* nRF5340
+* nRF9160
 
-When used on nRF9160, the driver behavior depends on whether it is used in a secure or non-secure application:
+When used in conjunction with the nRF52840, the entropy_cc3xx driver gathers entropy by using the CC310 hardware through the :ref:`nrf_cc3xx_platform_readme`.
+When used in conjunction with the nRF5340 or nRF9160, the driver behavior depends on whether it is used in a secure or non-secure application:
 
-When used on nRF5340, the driver behavior depends on whether it is used in a secure or non-secure application:
+* In a secure application, or when the :ref:`lib_spm` is not used, the entropy_cc3xx driver gathers entropy by using the CC3xx hardware through the :ref:`nrf_cc3xx_platform_readme`.
+* In a non-secure application, the driver gathers entropy through the :ref:`lib_secure_services` library.
 
-* When used :ref:`in a secure application <ug_nrf9160>`, or when the :ref:`lib_spm` is not used, the entropy_cc3xx driver gathers entropy by using the CC3xx hardware through the :ref:`nrf_cc3xx_platform_readme`.
-
-* When used :ref:`in a non-secure application <ug_nrf9160>`, the driver gathers entropy through the :ref:`lib_secure_services` library.
+For more details, see :ref:`ug_nrf5340` and :ref:`ug_nrf9160`.
```

Comment:
These are pretty generic references, and looking for entropy in both gives no results. Can the link be more specific?